service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -59,13 +65,19 @@ public boolean startsAfterToday() {
         return startDate.isAfter(ZonedDateTime.now(ZoneId.of("Europe/London")).toLocalDateTime());
```

Comment:
That's probably a tech debt story, but for stuff like this we should have our own Static class with its own clock. In this way we can have tests with stopped clock rather than using now(). Probably unlikely, but if the tests run at midnight, that could expose a flake

Reply:
we have it, see FixedTimeConfiguration, here it really does not matter and midnight should not afect as this operates on datetime and not on a date